src/components/ColorInputFlyout/ColorPickerFlyout.tsx

```diff
@@ -30,9 +31,9 @@ export const ColorPickerFlyout: FC<ColorPickerFlyoutProps> = ({
         <Flyout
             hug={false}
             onClick={onClick}
-            scrollable={false}
             onClose={onClose}
             isOpen={open}
+            fixedHeader={<ColorPreview color={currentColor || { r: 255, g: 255, b: 255 }} format={currentFormat} />}
```

Comment:
This was already fixed in: #549 
Please rebase

Reply:
That's from the rebase, but it had conflicts :)